libnavigation-util/src/main/java/com/mapbox/navigation/utils/internal/LoggerProvider.kt

```diff
@@ -26,58 +29,70 @@ internal object LoggerProvider {
 }
 
 /**
- * Alias of [LoggerProvider.logger]#v
+ * Alias of [com.mapbox.common.Logger.v]
  */
-fun logV(tag: String? = null, msg: String) {
+fun logV(msg: String, clzName: String? = null) {
```

Comment:
is `clzName` class name? could you add a doc on this param?

Reply:
Going to change it to `category` https://github.com/mapbox/mapbox-navigation-android/pull/5604#discussion_r834579885